pkg/kubelet/volume_host.go

```diff
@@ -131,6 +133,14 @@ func (kvh *kubeletVolumeHost) GetSubpather() subpath.Interface {
 	return kvh.kubelet.subpather
 }
 
+func (kvh *kubeletVolumeHost) GetCSIDriverLister() storagelister.CSIDriverLister {
+	return nil
```

Comment:
Kubelet also needs some logic to initialize the CSIDriverInformer, similar to the logic you added for AD controller [here](https://github.com/kubernetes/kubernetes/pull/75129/files#diff-72e66cba2ced392dc0f9cd10c424d6d0R342)

Reply:
Sure